WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -1467,6 +1468,10 @@ private extension OrderDetailsDataSource {
         return !paymentViewModel.hasBeenPartiallyCharged
     }
 
+    func isOrderCurrencyEligibleForCardPayment() -> Bool {
+        CurrencySettings.CurrencyCode(rawValue: order.currency) == .USD
```

Comment:
This will work for now, but we'll need the business folks to decide how this will work for non US merchants when we expand there. Please consider opening an issue for "how to determine supported currencies for in-person payments for non-US merchants".

Notably, support currencies are passed in the account response. See https://github.com/woocommerce/woocommerce-ios/blob/beaf197f6a326020cf83a747d53e989f2d862ae4/Networking/Networking/Model/WCPayAccount.swift#L16

Reply:
Sure #4928